eslint-rules/no-single-tests.js

```diff
@@ -1,10 +1,15 @@
+'use strict';
 module.exports = function(context) {
   return {
     Identifier: function(node) {
-      var nodeName = node.name;
+      // Only run these rules if it's in a test file.
+      if (context.getFilename().indexOf('test') === -1) {
```

Comment:
Driveby fix because one of the terminal methods is called `fit` which caused the lint to fail.